proxy/pkg/util/util.go

```diff
@@ -82,6 +86,11 @@ func InitScheduler() {
 	scheduler = gocron.NewScheduler(time.UTC)
 }
 
+func InitAccessReviewer(kConfig *rest.Config) (err error) {
+	accessReviewer, err = rbac.NewAccessReviewer(kConfig, nil)
+	return
```

Comment:
Can we avoid the named return? This is easy to get wrong when reading code.

Reply:
Fixed.